### ISSUE TITLE
Horizontal multi-target layout does not look nice with the new notebook

### ISSUE THREAD

**Issue by lopuhin** [TeamHG-Memex/eli5#178]
If there are a lot of targets, they might look poorly in the notebook, here is a screenshot from our CRF tutorial
![screenshot from 2017-05-02 16-01-24](https://cloud.githubusercontent.com/assets/424613/25618738/c5ee3166-2f50-11e7-9030-517d74ff1b9a.png)

The scroll is still there, but it's not enough.

I didn't check, but I think it didn't break like that in the previous jupyter notebook version - they clearly changed the CSS quite a bit.

**Comment by kmike** [comment]
Yeah, I also noticed that.
